packages/dependency-royalty-management/contracts/MultiRoyaltyDistributor.sol

```diff
@@ -125,4 +125,22 @@ abstract contract MultiRoyaltyDistributor is IEIP2981, IMultiRoyaltyDistributor,
         defaultRecipient[0] = Recipient({recipient: _defaultRoyaltyReceiver, bps: TOTAL_BASIS_POINTS});
         return defaultRecipient;
     }
+
+    function _setTokenRoyaltiesSplitter(uint256 tokenId, address payable splitterAddress) internal {
+        _tokenRoyaltiesSplitter[tokenId] = splitterAddress;
+        emit TokenRoyaltySplitterSet(tokenId, splitterAddress);
```

Comment:
Why skip natspec here?

Reply:
Do we have natspec on all the internal functions in our packages? If we do, then this should be covered here. 
As a minimum, we require natspec for external and public functions but if this is the only function we're not covering, and if it would be helpful to the reader to add natspec, we would ideally do so